### DISCUSSION TITLE
Opentelemetry

### DISCUSSION THREAD

**Discussion by ceramicv** [danielmiessler/Fabric#897] [General]
When I issued the go install github.com/danielmiessler/yt@latest to install yt, I noticed that it downloaded a bunch of opentelemetry files - what are these being used for? ::

go: downloading github.com/danielmiessler/yt v0.0.0-20240816204735-a908bc4ed8b6
go: downloading github.com/anaskhan96/soup v1.2.5
go: downloading go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp v0.49.0
go: downloading golang.org/x/sys v0.22.0
go: downloading go.opentelemetry.io/otel v1.24.0
go: downloading go.opentelemetry.io/otel/trace v1.24.0
go: downloading go.opentelemetry.io/otel/metric v1.24.0

**Comment by davejpeters** [comment]
GO has telemetry installed in the language by default, though I believe it is only active if it is declared active.
